sdk/storage/azure-storage-blob-nio/src/main/java/com/azure/storage/blob/nio/AzureFileSystemProvider.java

```diff
@@ -181,11 +195,111 @@ public SeekableByteChannel newByteChannel(Path path, Set<? extends OpenOption> s
     }
 
     /**
+     * The existence of a directory in the {@code AzureFileSystem} is defined on two levels. Weak existence is defined
```

Comment:
Reviewers, please read this Javadoc carefully and ensure we are ok with the content and clarity.

Reply:
It might be helpful to highlight some of the key words like strong existence, marker blob, etc in like bold/italics?